_config.yml

```diff
@@ -14,8 +14,8 @@ plugins:
   - jekyll-feed
   - jekyll-redirect-from
 
-sass:
-  sass_dir: docs/1.0/assets/scss
+# sass:
+#   sass_dir: docs/1.0/assets/scss
```

Comment:
This can and will be removed before `merging` into `master` and `gh-pages` branches. It's used for Jekyll but node-sass is handling our CSS compilation.